.github/workflows/simorgh-deploy-storybook.yml

```
@@ -0,0 +1,40 @@
+name: Deploy Storybook
+
+# Run this workflow every time a new commit pushed to your repository
+on:
+  push:
+    branches:
+      - deploy-storybook-gh-action
```

Reminder to update this to correct branch before we merge
```suggestion
      - latest
```